src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -41,3 +43,38 @@ export const human38stats = {
     structural_variant_count: 6638628
   }
 };
+
+type SpeciesSidebarData = {
+  [genomeId: string]: SpeciesSidebarPayload;
+};
+
+export const sidebarData: SpeciesSidebarData = {
+  homo_sapiens_GCA_000001405_28: {
+    species: {
+      display_name: 'Human',
+      scientific_name: 'Homo sapiens'
+    },
+    assembly: {
+      name: 'GRCh38.p13',
+      source: {
+        name: 'INSDC Assembly',
+        id: 'GCA_000001405.28',
+        url: ''
+      },
+      level: 'complete genome'
+    },
+    annotation: {
+      provider: 'Ensembl',
+      method: 'Full genebuild',
+      last_updated_date: 'Feb 2019',
+      gencode_version: 'GENCODE 34',
+      database_version: '100.38',
+      taxonomy_id: '9606'
+    },
+    psuedoautosomal_regions: {
+      description: `The Y chromosome in this assembly contains two psuedoautosomal regions (PARs) that were taken from the corresponding X chromosoles and are exact duplicates:
+        chrX:10,000-2,781,479 = chrY:10,000-2,781,479
+        chrX:155,701,382-156,030,895 = chrY:56,887,02-57,217,415 `
+    }
```

Comment:
As Andy said, the notice about pseudoautosomal regions is relevant only for a single species, which is human. I'm thinking that the responsibility for showing or hiding this notice may be hard-coded into the sidebar itself.

Unless we can generalise this block to `notes` or something, if some other species are also going to have some unique text blocks about them. We'll need to ask Andrea / Steve what they think.